### ISSUE TITLE
Feature request: Helmet Upgrade - Pumpkin Mode

### ISSUE THREAD

**Issue by lafflam** [TeamPneumatic/pnc-repressurized#369]
Feature Request:

This is a suggestion for a new helmet upgrade idea. Please consider creating a helmet upgrade that includes pumpkin(s) in the recipe. When applied to the helmet, it obscures the vision of the player just like wearing a regular pumpkin and allows the player to look at Endermen without provoking them. Please provide a configurable hotkey so this feature can be toggled at will.

I think for game balance, the vision should remain obscured when Pumpkin Mode is enabled. However, you could provide an option in the config file that removes the obscured vision so that players can decide if they'd rather have clear vision or not.

**Comment by desht** [comment]
Yeah, that's quite a neat idea.  TBH I would simply have it obscure the player's vision but disable enderman aggro, with no possibility of disabling the obscuring. That's balanced, especially if the feature can be easily toggled on and off.

This may not make it into the 1.12.2 version (simple due to the amount of time I don't have right now...), but certainly under consideration for 1.14.x.

**Comment by lafflam** [comment]
Glad you like the idea. I totally understand the need to prioritize 1.14 over 1.12. That said, if you suddenly find yourself with nothing to do... (hint hint for 1.12). Otherwise, I suppose I could pester @pyure for a 1.14 version of the InfiTech modpack :-D

Peace.

**Comment by TeamSpen210** [comment]
Looks like this is implemented now :)

**Comment by desht** [comment]
It is, although in the so-far unreleased 1.18 version...
